backend/dissemination/search.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import Q
+
+from dissemination.models import General
+
+
+def search_general(
+    names=None,
+    uei_or_eins=None,
+    start_date=None,
+    end_date=None,
+    cog_or_oversight=None,
+    agency_name=None,
+    audit_years=None,
+):
+    query = Q(is_public=True)
+
+    # TODO: use something like auditee_name__contains
+    # SELECT * WHERE auditee_name LIKE '%SomeString%'
+    if names:
```

Comment:
Just checking: this is still the exact match query?

Reply:
yes